### PR TITLE
install cython automatically as a dependency

### PR DESCRIPTION
**Pull request by karlind**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import platform
 import sys
 from io import open  # for Python 2 (identical to builtin in Python 3)
-from setuptools import Extension, find_packages, setup
+from setuptools import Extension, find_packages, setup, dist
+
+dist.Distribution().fetch_build_eggs(['Cython'])
```
Comment (hellock) [inline setup.py]: `numpy` may also be included..

Comment (karlind) [reply setup.py]: `numpy` has been included

**Comment by hellock** [comment]
Thanks for the fix! Could you check the linting and formatting errors?

**Comment by karlind** [comment]
Actually I have already checked the errors.
The errors say,
```
$ flake8
./setup.py:8:1: E402 module level import not at top of file
./setup.py:9:1: E402 module level import not at top of file
The command "flake8" failed and exited with 1 during .
```
Because `dist.Distribution().fetch_build_eggs(['Cython', 'numpy>=1.11.1'])` must be exectued before `import`, so it seems not easy to fix these linting errors.

**Comment by karlind** [comment]
May be we can ignore such errors by using
```
flake8 --ignore=E402 ./setup.py
```

**Comment by hellock** [comment]
> May be we can ignore such errors by using
> 
> ```
> flake8 --ignore=E402 ./setup.py
> ```

Use inline comments instead.

```python
import numpy as np  # noqa: E402
```

**Pull request merged**